app/config/services.yml

```diff
@@ -218,6 +219,7 @@ services:
 
     Wallabag\Helper\HttpClientFactory:
         calls:
+            - ['setUserAgent', ["Wallabag (Guzzle/5)"]]
```

Comment:
having the user agent as a parameter, you could inject the parameter directly to the HttpClientFactory service, no need for the setter